src/Exception/NotFoundException.php

```diff
@@ -8,6 +8,6 @@
 /**
  * No entry was found in the container.
  */
-interface NotFoundException extends ContainerException
+interface NotFoundException
```

Comment:
@mnapoli I would have missed in the meta, why these exceptions don't extend the \Exception class? Is it due to the PHP 7 throwable coming to play?